src/pages/find-offers/FindOffers.jsx

```diff
@@ -29,6 +29,12 @@ const FindOffers = () => {
   const onBookmarkClick = (id) => {console.log(id)}
   const handleToggleOpenFilters = () => setOpenFilters(prev => !prev)
   const handleShowingTutorOffers = () => setShowingTutorOffers(prev => !prev)
+
+  const currentOffersOnPage = useMemo(() => {
+    const firstPageNumber = (currentPage - 1) * mockDataPagination.pageSize
+    const lastPageNumber = firstPageNumber + mockDataPagination.pageSize
+    return mockOffers.slice(firstPageNumber, lastPageNumber)
+  }, [currentPage, mockDataPagination.pageSize])
   
```

Comment:
Please, name variables correctly.  firstPageNumber and lastPageNumber don't consist of pagination. It is hard to understand.

Reply:
fixed